<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

	<!-- EKF -->
	<include file="$(find localization)/launch/sub_launch/ekf.launch"/>

	<!-- REMAPPING/FILTERING -->
	<param name="odometry_scale" value="0.01"/>
	<param name="gps_ma_samples" value="5"/>
	<node name="odom_cvt" pkg="localization" type="odom_cvt.py"/>
	<include file="$(find localization)/launch/sub_launch/navsat_transform.launch"/>

	<remap from="/imu/imu/data" to="/imu/data"/> <!-- Whoops -->

	<!-- TRANSFORMS -->
	<include file="$(find localization)/launch/sub_launch/hector_trajectory.launch"/>
	<node pkg="tf2_ros" type="static_transform_publisher" name="odom" args="0 0 0 0 0 0 world odom" />
	<node pkg="tf2_ros" type="static_transform_publisher" name="imu" args="0.3302 0 0.7112 			1.57 0 0 base_link imu_link" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="gps" args="0.8382 0 1.5494 				0 0 0 base_link /gps" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="velodyne" args="1.1684 0 1.6256 	0 0 0 base_link lidar_link" />

	<!-- PLAYBACK -->
	<node pkg="rosbag" type="play" name="rosbagplay" args="$(find localization)/bags/05.bag --clock -d 5 -r 1" required="true"/>

		<!-- <node pkg="gps_common" type="utm_odometry_node" name="gps_common">
			<param name="child_frame_id" value="gps_odom"/>
		 	<param name="frame_id" value="odom"/>
		</node> -->

		<!--VISUALIZATION -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find localization)/launch/sub_launch/localization.rviz"/>

</launch>
